gobblin-temporal/src/main/java/org/apache/gobblin/temporal/ddm/workflow/impl/NestingExecOfProcessWorkUnitWorkflowImpl.java

```diff
@@ -41,7 +41,7 @@ public class NestingExecOfProcessWorkUnitWorkflowImpl extends AbstractNestingExe
       .build();
 
   private static final ActivityOptions ACTIVITY_OPTS = ActivityOptions.newBuilder()
-      .setStartToCloseTimeout(Duration.ofSeconds(999))
+      .setStartToCloseTimeout(Duration.ofMinutes(20))
```

Comment:
We probably want to make this configurable, add a TODO? Otherwise there can be unforseen behavior

Reply:
I definitely agree on configurability.  I added a TODO [over there](https://github.com/apache/gobblin/pull/3886/files#diff-8b215b0860cefa8c3d067f90482ea09e09bfab64ede78e369c863240e0c28dfaR55) and can add one here too